OpenRA.Mods.Common/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -43,6 +47,9 @@ public WorldInteractionControllerWidget(World world, WorldRenderer worldRenderer
 		{
 			World = world;
 			this.worldRenderer = worldRenderer;
+			ChromeMetrics.TryGet("AltSelectionColor", out altSelectionColor);
```

Comment:
Sorry, one last nit that I should have mentioned last time: we should make sure to default these to white if they aren't defined in the metrics.

Reply:
Fixed. When undefined they will now default to white.